gRPC/Client Examples/RealtimeGraphing/BICManager.cs

```diff
@@ -150,6 +153,24 @@ public bool BICConnect()
             Console.WriteLine("Connecting to implantable device.");
             var connectDeviceReply = deviceClient.ConnectDevice(new ConnectDeviceRequest() { DeviceAddress = DeviceName, LogFileName = "./deviceLog.txt" });
 
+            // Perform an impedance check if requested
+            if (runImpTest)
```

Comment:
Why is the impedance test being added as part of initialization and not as it's own protobuf-defined microservice function?